Jellyfin.ApiClient/Model/Querying/ArtistsQuery.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Jellyfin.ApiClient.Model.Querying
+{
+    /// <summary>
+    /// Class ArtistsQuery
+    /// </summary>
+    public class ArtistsQuery : ItemsByNameQuery
+    {
+        /// <summary>
+        /// Filter by artists that are on tour, or not
+        /// </summary>
+        /// <value><c>null</c> if [is on tour] contains no value, <c>true</c> if [is on tour]; otherwise, <c>false</c>.</value>
+        public bool? IsOnTour { get; set; }
+    }
```

Comment:
Now that I've seen the code I don't think you'll need this. We don't track artists on tour at all.

Reply:
As I implement actual client functionality I'll be weeding out dead API stuff, so this will likely be removed in a future PR